src/utils/useLocationUpdateEffect.ts

```diff
@@ -1,188 +0,0 @@
-import { getCurrentTimeInMilliseconds } from "./time";
-import { openRoomUrl, openUrl, venueInsideUrl } from "./url";
-
-const LOCATION_INCREMENT_SECONDS = 10;
-const LOCATION_INCREMENT_MS = LOCATION_INCREMENT_SECONDS * 1000;
-
-export const updateLocationData = (
-  user: UserInfo,
-  locationName: { [key: string]: number },
-  lastSeenIn: { [key: string]: number } | undefined
-) => {
-  const location = locationName ?? {};
-  const prevRoomName =
-    lastSeenIn &&
-    Object.keys(lastSeenIn).find((lastSeen) => lastSeen.includes("/"));
-
-  if (lastSeenIn && prevRoomName) {
-    delete lastSeenIn[prevRoomName];
-  }
-
-  const roomVenue =
-    locationName && Object.keys(locationName).length
-      ? Object.keys(locationName)[0]
-      : null;
-
-  updateUserProfile(user.uid, {
-    lastSeenAt: getCurrentTimeInMilliseconds(),
-    lastSeenIn:
-      !locationName && !lastSeenIn
-        ? null
-        : lastSeenIn
-        ? { ...lastSeenIn, ...location }
-        : location,
-    room: !locationName && !lastSeenIn ? null : roomVenue,
-  });
-};
-
-// get Profile from the firebase
-// @debt rename this trackLocationEntered?
-export const trackRoomEntered = (
-  user: UserInfo,
-  locationName: { [key: string]: number },
-  lastSeenIn: { [key: string]: number } | undefined
-) => {
-  updateLocationData(user, locationName, lastSeenIn);
-};
-
-export interface TrackRoomEnteredNGProps {
-  user?: UserInfo;
-  venue: AnyVenue;
-  room: AnyRoom;
-  lastSeenIn?: Record<string, number>;
-}
-
-export const trackRoomEnteredNG = ({
-  user,
-  venue,
-  room,
-  lastSeenIn,
-}: TrackRoomEnteredNGProps) => {
-  if (!user) return;
-
-  trackRoomEntered(
-    user,
-    { [`${venue.name}/${room.title}`]: getCurrentTimeInMilliseconds() },
-    lastSeenIn
-  );
-};
-
-export interface TrackVenueEnteredProps {
-  user?: UserInfo;
-  venue: AnyVenue;
-  lastSeenIn?: Record<string, number>;
-}
-
-export const trackVenueEntered = ({
-  user,
-  venue,
-  lastSeenIn,
-}: TrackVenueEnteredProps) => {
-  if (!user) return;
-
-  trackRoomEntered(
-    user,
-    { [venue.name]: getCurrentTimeInMilliseconds() },
-    lastSeenIn
-  );
-};
-
-export const leaveRoom = (user: UserInfo) => {
-  updateUserProfile(user.uid, {
-    lastSeenAt: 0,
-    lastSeenIn: {},
-    room: null,
-  });
-};
-
-export interface BaseEnterRoomWithCountingProps {
-  user?: UserInfo;
-  profile?: User;
-  venue: WithId<AnyVenue>;
-}
-
-export interface EnterRoomWithCounting extends BaseEnterRoomWithCountingProps {
-  room?: AnyRoom;
-}
-
-export const openRoomWithCounting = ({
-  user,
-  profile,
-  venue,
-  room,
-}: EnterRoomWithCounting) => {
-  if (!room) {
-    trackVenueEntered({
-      user,
-      venue,
-      lastSeenIn: profile?.lastSeenIn,
-    });
-
-    openUrl(venueInsideUrl(venue.id));
-    return;
-  }
-
-  // Track room counting
-  trackRoomEnteredNG({
-    user,
-    venue,
-    room,
-    lastSeenIn: profile?.lastSeenIn,
-  });
-
-  openRoomUrl(room.url);
-};
```

Comment:
Why was this removed? The seemed equivalent function in the new code `enterVenue` doesn't seem to do any tracking. This feels like a regression.

Reply:
You don't need tracking for entering the venue inside the platform. We have `VenuePage` that has us covered there

Reply:
Makes sense.